.scripts/copyBitsToAzureBlob.ps1

```diff
@@ -23,4 +23,8 @@ azcopy copy "$JarPath/$SDKVersionNumber/applicationinsights-agent-$SDKVersionNum
 azcopy copy "$JarPath/$SDKVersionNumber/applicationinsights-agent-$SDKVersionNumber-javadoc.jar" "https://azuresdkpartnerdrops.blob.core.windows.net/drops/applicationinsights-sdk/java/$SDKVersionNumber/"
 azcopy copy "$JarPath/$SDKVersionNumber/applicationinsights-agent-$SDKVersionNumber-sources.jar" "https://azuresdkpartnerdrops.blob.core.windows.net/drops/applicationinsights-sdk/java/$SDKVersionNumber/"
 azcopy copy "$JarPath/$SDKVersionNumber/applicationinsights-agent-$SDKVersionNumber.pom" "https://azuresdkpartnerdrops.blob.core.windows.net/drops/applicationinsights-sdk/java/$SDKVersionNumber/"
+azcopy copy "$JarPath/$SDKVersionNumber/applicationinsights-runtime-attach-$SDKVersionNumber.jar" "https://azuresdkpartnerdrops.blob.core.windows.net/drops/applicationinsights-sdk/java/$SDKVersionNumber/"
+azcopy copy "$JarPath/$SDKVersionNumber/applicationinsights-runtime-attach-$SDKVersionNumber-javadoc.jar" "https://azuresdkpartnerdrops.blob.core.windows.net/drops/applicationinsights-sdk/java/$SDKVersionNumber/"
+azcopy copy "$JarPath/$SDKVersionNumber/applicationinsights-runtime-attach-$SDKVersionNumber-sources.jar" "https://azuresdkpartnerdrops.blob.core.windows.net/drops/applicationinsights-sdk/java/$SDKVersionNumber/"
+azcopy copy "$JarPath/$SDKVersionNumber/applicationinsights-runtime-attach-$SDKVersionNumber.pom" "https://azuresdkpartnerdrops.blob.core.windows.net/drops/applicationinsights-sdk/java/$SDKVersionNumber/"
```

Comment:
The _applicationinsights-runtime-attach_ files (jar, sources, pom) can be generated today?